torch/csrc/jit/fusers/cpu/annotated_graph.h

```diff
@@ -0,0 +1,29 @@
+#if !(defined _WIN32)
+#pragma once
+
+#include "torch/csrc/jit/fusers/fuser_interface.h"
+
+#include "torch/csrc/WindowsTorchApiMacro.h"
+
+#include "torch/csrc/jit/ir.h"
+
+#include <vector>
+
+namespace torch { namespace jit { namespace cpufuser {
+
+TORCH_API struct AnnotatedGraph {
+  // short-term storage only, so it borrows Graph.
+  AnnotatedGraph(Graph& graph, int device)
+  : graph(&graph), device(device) {}
+
+  Graph* graph = nullptr;
+  int device = kCPUDevice;
+  std::vector<TensorDesc> input_desc;
+  std::vector<TensorDesc> output_desc;
+};
```

Comment:
Is this really CPU specific in any way? It looks like this one, `TensorDesc` and a few other structs are shared and we're unnecessarily duplicating code. Can you move them as a shared util to the `fusers` directory?